lib/beaker/host.rb

```diff
@@ -499,6 +501,7 @@ def do_rsync_to from_path, to_path, opts = {}
     'freebsd',
     'windows',
     'pswindows',
+    'eos',
```

Comment:
I mean... "kind of".  So, the thing there is that those files are declaring subclasses of `Beaker::Host` so ideally that base class is declared before they are encountered.

Various ways to do that:

 - Do what they did -- declare the full base class, all the code, etc., and load those classes after it, from the same file.
 - Declare the base class, and don't require the subclasses in the same file, relying on them to be included later, autoloaded, etc.
 - Declare the base class. Inside the base class, when something is actually instantiated (via a call to `.new` aka `initialize`; or on module inclusion if it's a module) go and require the relevant classes. This is often "require all the classes in this whole directory".
 - Declare the base class, require those files inside the declaration.
 - Declare the base class, and once that's declared, close it, require those files, and, then re-open the class.

So, there's a bunch of ways to do it (probably 3 or 4 more I didn't think of), and I agree, dropping those at the end of the file is less than ideal.